pom.xml

```diff
@@ -252,6 +254,21 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <artifactId>maven-deploy-plugin</artifactId>
```

Comment:
I've added a possibility of skipping maven artifact upload, since we don't need it to upload the image.